sdk/python/testvectors/nem/multisig_account_modification.py

```diff
@@ -45,6 +45,31 @@
 				}
 			],
 			'min_approval_delta': 0x1
+		},
+		# comment: v2, add, add, del, increase min-approval (out of order)
+		{
+			'type': 'multisig_account_modification_transaction',
+			'modifications': [
+				{
+					'modification': {
+						'modification_type': 'add_cosignatory',
+						'cosignatory_public_key': '5D378657691CAD70CE35A46FB88CB134232B0B6B3655449C019A1F5F20AE9AAD'
+					}
+				},
+				{
+					'modification': {
+						'modification_type': 'add_cosignatory',
+						'cosignatory_public_key': 'D79936328C188A4416224ABABF580CA2C5C8D852248DB1933FE4BC0DCA0EE7BC'
+					}
+				},
+				{
+					'modification': {
+						'modification_type': 'delete_cosignatory',
+						'cosignatory_public_key': 'BCEAE8AC9F6630893C75381CFA44BA971E556EA245C851D5CB224A18D6871843'
+					}
+				}
```

Comment:
wait, _how_ exactly are they unsorted? can you add some comment right above it?

(you can add testnet addresses obtained from those keys)

Reply:
because sorting is done by address